src/components/action-popover/action-popover.style.ts

```diff
@@ -19,12 +20,134 @@ const Menu = styled.div`
     `${theme.zIndex?.popover}`}; // TODO (tokens): implement elevation tokens - FE-4437
 `;
 
+type variants = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9;
+
+function getLeftPaddingValues(variant: variants) {
+  if (variant === 1) {
+    return "var(--spacing100)";
+  }
+  if (variant === 2) {
+    return "var(--spacing400)";
+  }
+  if (variant === 3 || variant === 4) {
```

Comment:
This is what I mean whereby you don't really need separate variants for `3` and `4` or `5`,`6` and `7` as each one produces the same padding left or right token value